Create promoting integrations (#176)
mautic/mautic-community-handbook: pages/06.policies/09.Promoting-integrations/docs.md

2. Use Cases:

- Projects that demonstrate a clear alignment with Mautic's use cases and goals will be given priority.

- The Marketing Team will assess how well a project addresses the needs of Mautic users and enhances their experience.
3. Members:

- Projects that are members of Mautic will be given priority over those that are not.

# Maintenance Responsibilities

The maintenance of projects listed on the Mautic website home page is the responsibility of the Marketing Team. This ensures consistency, accuracy, and a seamless experience for visitors.

1. Content Updates:

- The Marketing Team will regularly review and update the information related to listed projects to reflect their current status, features, and any relevant changes.

- Project maintainers are encouraged to communicate updates to the Marketing Team for timely inclusion.

2. Removal of Inactive Projects:

- Inactive projects, or those no longer aligning with Mautic's goals, may be removed from the home page to maintain relevance.

- The Marketing Team will communicate with project maintainers before removal, providing an opportunity for clarification or updates.

# Priority for Open Source Projects

> Projects with an open source nature are given priority for listing on the Mautic website home page. Open source aligns with the collaborative and community-driven spirit of Mautic. However, partner projects may also be considered based on their significance and value to the Mautic ecosystem.

Reviewer: ```suggestion
Projects with an open source nature are given priority for listing on the Mautic website home page. Open source aligns with the collaborative and community-driven spirit of Mautic. However, projects from official [Mautic Partners](https://mau.tc/partners) may also be considered based on their significance and value to the Mautic ecosystem.
```

1. Open Source Priority:

- Open source projects are favored for inclusion due to their accessibility, transparency, and collaborative nature.

- The Marketing Team encourages the community to actively contribute to Open Source projects.

> 2. Partner Projects:

Reviewer: @tobsowo what do you mean by 'Partner project's here? I'm a bit concerned it might be confused with our 'Mautic Partners'. Could you maybe explain what you mean so we can word it better? Thanks!

Author: Partners has to do with the official community partners

Reviewer: Ah awesome - I'll make sure that's worded clearer then!

> - Partner projects, while not open source, may be considered if they provide substantial value to the Mautic community.

Reviewer: ```suggestion
- Projects from official Mautic Partners may be considered, even if not open source, if they provide substantial value to the Mautic community.
```

---
title: 'Policy on promoting integrations with Mautic'
twitterenable: true
twittercardoptions: summary
articleenabled: false
orga: {  }
personenabled: false
facebookenable: true
menu: 'Promoting integrations'
taxonomy:
    category:
        - docs
---

# Introduction

The Mautic website serves as a hub for the Mautic community, showcasing innovative projects that align with our mission and values. The purpose of this policy is to provide clarity on the criteria and process for listing projects and integrations on the Mautic website's home page.

# Criteria for Inclusion

Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.

1. Popularity:

> - Projects that have gained popularity within the Mautic community are more likely to be featured on the home page.

Reviewer: ```suggestion
- Projects that have gained popularity within the Mautic community are more likely to be featured on the home page, as members are more likely to vote for them.
```

> - Popularity may be determined by factors such as community engagement, user feedback, and adoption rates.

Reviewer: ```suggestion
```

Reviewer: This is far too subjective and open to bias. I think we should instead allow the community to direct this.

> 

Reviewer: ```suggestion
```

2. Use Cases:

- Projects that demonstrate a clear alignment with Mautic's use cases and goals will be given priority.

> - The marketing team will assess how well a project addresses the needs of Mautic users and enhances their experience.

Reviewer: ```suggestion
- The Marketing Team will assess how well a project addresses the needs of Mautic users and enhances their experience.
```

3. Members:

- Projects that are members of Mautic will be given priority over those that are not.

# Maintenance Responsibilities

> The maintenance of projects listed on the Mautic website home page is the responsibility of the marketing team. This ensures consistency, accuracy, and a seamless experience for visitors.

Reviewer: ```suggestion
The maintenance of projects listed on the Mautic website home page is the responsibility of the Marketing Team. This ensures consistency, accuracy, and a seamless experience for visitors.
```

1. Content Updates:

> - The marketing team will regularly review and update the information related to listed projects to reflect their current status, features, and any relevant changes.

Reviewer: ```suggestion
- The Marketing Team will regularly review and update the information related to listed projects to reflect their current status, features, and any relevant changes.
```

- Project maintainers are encouraged to communicate updates to the marketing team for timely inclusion.

2. Removal of Inactive Projects:

- Inactive projects, or those no longer aligning with Mautic's goals, may be removed from the home page to maintain relevance.

> - The marketing team will communicate with project maintainers before removal, providing an opportunity for clarification or updates.

Reviewer: ```suggestion
- The Marketing Team will communicate with project maintainers before removal, providing an opportunity for clarification or updates.
```

# Priority for Open Source Projects

Projects with an open source nature are given priority for listing on the Mautic website home page. Open source aligns with the collaborative and community-driven spirit of Mautic. However, partner projects may also be considered based on their significance and value to the Mautic ecosystem.

1. Open Source Priority:

- Open source projects are favored for inclusion due to their accessibility, transparency, and collaborative nature.

> - The marketing team encourages the community to actively contribute to open source projects.

Reviewer: ```suggestion
- The Marketing Team encourages the community to actively contribute to Open Source projects.
```

---
title: '<your title>'
twitterenable: true
twittercardoptions: summary
articleenabled: false
orga: {  }
personenabled: false
facebookenable: true
menu: 'Promoting integrations'
taxonomy:
    category:
        - docs
---

# Introduction

The Mautic website serves as a hub for the Mautic community, showcasing innovative projects that align with our mission and values. The purpose of this article is to provide clarity on the criteria and process for listing projects and integrations on the Mautic website's home page.

# Criteria for Inclusion

> Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.

Author: The three were picked at random.
The point on quaterly poll is a good suggestion and we can do that.

> Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.

Reviewer 2: @RCheesley I can speak for CyberPanel. We were one of the first control panels who integrated Mautic one-click installation. 

Not only that we promoted and brought many users to Mautic. Then we've created lots of study material which have been seen by many people

https://www.youtube.com/watch?v=zeUfl0G6uOo&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D
https://www.youtube.com/watch?v=ML6NAE2h0xY&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D

> Projects featured on the Mautic website home page are selected based on their popularity and relevance to Mautic users. The decision to list a project is primarily made by the marketing team, considering factors such as the project's impact, use cases, and overall community value.

Reviewer: ```suggestion
Projects featured in the Mautic Integrations Directory must be proposed by a member of Mautic via the [Community Portal](https://community.mautic.org/assemblies/marketing-team/f/53/proposals/new). Integrations require a minimum of five votes from the community to be considered for inclusion, and be endorsed by the Marketing Team. 

The projects listed on the Mautic website home page are selected based on their popularity and relevance to Mautic users, which is accomplished with a quarterly vote on the Community Portal of all eligible integrations. Members of Mautic are eligible to vote. 
```

@tobsowo I mean this should be documented. I believe we should use the community voting option. This is less open to personal biases and gives the community the opportunity to be engaged in the process and therefore it be more representative of sentiment. I'd suggest we limit it to members of Mautic only who are allowed to both propose integrations and vote. If quarterly proves to be too frequent we could make it 6-monthly.

I can help setting up the voting process and documenting it for you so that it's easy for you to reproduce every 3 months.
